src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String randomColor, Random random) {
+    }
```

Comment:
The constructor `Ball(String randomColor, Random random)` is not used and seems unnecessary given the current implementation. Consider removing it if it's not required for your application.